src/main/java/com/flightstats/hub/spoke/SpokeContentDao.java

```diff
@@ -20,6 +26,16 @@
 public class SpokeContentDao {
     private static final Logger logger = LoggerFactory.getLogger(SpokeContentDao.class);
 
+    static final String GET_OLDEST_ITEM_COMMAND = "find %s -type f -printf '%%T+ %%p\\n' | sort | head -n 1";
```

Comment:
Trying to think whether you need "sort -n" here.

Reply:
What would `sort -n` get me that normal `sort` isn't? The `find` is returning lines that look like this:

```
YYYY-MM-DD+HH:MM:SS.SSSSSSSSSS /mnt/spoke/store/channel/yyyy/mm/dd/hh/mm/ssSSShash
```

I read this from the man pages but I'm still not sure:

>Comparisons are based on one or more sort keys extracted from each line of input, and are performed lexicographically, according to the current locale's collating rules and the specified command-line options that can tune the actual sorting behavior.  By default, if keys are not given, sort uses entire lines for comparison.

>-n, --numeric-sort, --sort=numeric
>             Sort fields numerically by arithmetic value.  Fields are supposed to have optional blanks in the beginning, an optional minus sign, zero or more digits (including decimal point and possible thousand separators).